src/schemas/40-10007/schema.js

```diff
@@ -0,0 +1,171 @@
+import constants from '../../common/constants';
+import originalDefinitions from '../../common/definitions';
+import schemaHelpers from '../../common/schema-helpers';
+import _ from 'lodash';
+
+let definitions = _.cloneDeep(originalDefinitions);
+let modifiedToursOfDuty = definitions.toursOfDuty;
+modifiedToursOfDuty.items.required = ['serviceBranch'];
+delete modifiedToursOfDuty.items.properties.benefitsToApplyTo;
+delete modifiedToursOfDuty.items.properties.applyPeriodToSelected;
+delete modifiedToursOfDuty.items.properties.serviceStatus;
+
+_.merge(modifiedToursOfDuty, {
+  items: {
+    properties: {
+      dischargeType: {
+        type: 'string',
+        'enum': constants.dischargeTypes.map(option => option.value)
+      },
+      highestRank: {
+        type: 'string'
+      }
+    }
+  }
+});
+
+let schema = {
+  $schema: 'http://json-schema.org/draft-04/schema#',
+  title: 'APPLICATION FOR PRE-NEED DETERMINATION OF ELIGIBILITY IN A VA NATIONAL CEMETERY',
+  type: 'object',
+  additionalProperties: false,
+  definitions: _.pick(definitions, [
+    'address',
+    'date',
+    'dateRange',
+    'files',
+    'fullName',
+    'gender',
+    'phone',
+    'ssn',
+    'vaFileNumber'
+  ]),
+  properties: {
+    applications: {
+      type: 'array',
+      items: {
+        type: 'object',
+        properties: {
+          applicant: {
+            type: 'object',
+            properties: {
+              applicantEmail: { type: 'string', format: 'email' },
+              applicantPhoneNumber: schemaHelpers.getDefinition('phone'),
+              applicationRelationshipToClaimant: {
+                type: 'string',
+                'enum': [
+                  'self',
+                  'Authorized Agent/Rep'
+                ]
+              },
+              completingReason: { type: 'string' },
+              mailingAddress: schemaHelpers.getDefinition('address'),
+              name: schemaHelpers.getDefinition('fullName')
```

Comment:
EOAS has a `maidenName` field as well (currently max 15 characters, not required)